.github/workflows/frontend.yml

```
@@ -81,26 +85,68 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
+      - name: go mod cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}  
+          restore-keys: |
+            ${{ runner.os }}-go-mod-
+      - name: go build cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+          key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-build-
```

so we dont cache the smaller build cache from the mock server for the regular build
```suggestion
          key: ${{ runner.os }}-go-mock-build-${{ hashFiles('**/go.sum') }}
          restore-keys: |
            ${{ runner.os }}-go-build-
            ${{ runner.os }}-go-mock-build-
```